### PR TITLE
refactor: Rename 'syndesis-restricted' to 'syndesis', removed docker image mode

### PR DESCRIPTION
**Pull request by rhuss**

### CODE REVIEW

**Comment by paoloantinori** [comment]
@rhuss are you also planning to update the corresponding quickstart on syndesis.io ?

https://github.com/syndesisio/syndesis.io/blob/master/content/quickstart.md

**Comment by rhuss** [comment]
yes, need to find out all the places where the 'restricted' suffix has been used. Guess its in some documentation as well (and for sure in the deployment scripts of our cluster)

File: app/deploy/README.md [hunk 1]
```diff
@@ -108,23 +103,25 @@ OpenShift includes the ability for a service account to act as a limited OAuthCl
...
 ```
 
 #### Create the template to use
 
-We will create the template in the project, rather than in the openshift namespace as it is assumed the user does not have cluster-admin rights:
+Create the template:
 
 ```bash
-$ oc create -f https://raw.githubusercontent.com/syndesisio/syndesis-openshift-templates/master/syndesis-dev-restricted.yml
+$ oc create -f syndesis.yml
```
Comment (zregvart) [inline app/deploy/README.md]: Should this point to raw.githubusercontent.com...?

Comment (dsimansk) [reply app/deploy/README.md]: It refers to local file in git repo, imho.

Comment (rhuss) [reply app/deploy/README.md]: Yes, its just meant as an example. wanted to get rid of the URL which is bogus anyway and I just assume that you cd is the one where the README is.

And tbh, we should maybe even remove this doc here and _only_ reference to the manual.

**Pull request merged**